### PR TITLE
Changed Menu components > now use MenuButton inside MenuItem

### PR DESCRIPTION
**Pull request by chrisvanmook**
## Amsterdam styled components pull request

Changed:
- Added: MenuButton component: this now can be used as a button or link inside MenuItem
- Changed: **BREAKING** MenuItem now doesn't accept any onClick or href props: it's just a wrapper 
  now. Use MenuButton inside MenuItem
- Removed: functionality to use arrows inside a MenuFlyOut / MenuToggle. This created a lot of 
  (unnecessary) complexity. Perhaps will be added in the future if really needed. 

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-ui/src/components/Button/ButtonStyle.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ const getVariant = () => ({
     case 'blank':
       return css`
         ${svgFill('tint', 'level7')}
+        background-color: ${getColor('tint', 'level1')({ theme })};
```
Comment (royderks) [inline packages/asc-ui/src/components/Button/ButtonStyle.ts]: Does the `getColor` function has to curry the `theme`? As `svgFill` doesn't seem to do so

File: packages/asc-ui/src/components/Menu/MenuButton/MenuButton.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  MenuButtonTextWrapperStyle,
+  Props,
+} from './MenuButtonStyle'
+
+const MenuButton: React.FC<
+  Props &
+    React.ButtonHTMLAttributes<HTMLButtonElement> &
+    React.AnchorHTMLAttributes<HTMLAnchorElement>
+> = ({ children, ...otherProps }) => (
+  <MenuButtonStyle iconSize={14} variant="blank" {...otherProps}>
```
Comment (royderks) [inline packages/asc-ui/src/components/Menu/MenuButton/MenuButton.tsx]: Do we export the variants somewhere as constants?

Comment (chrisvanmook) [reply packages/asc-ui/src/components/Menu/MenuButton/MenuButton.tsx]: not now, have to figure out how to do this in combination with enums... leave it out of scope for now

**Comment by royderks** [comment]
Hover style is missing here
Screenshot 2019-08-13 at 10 04 15

**Pull request merged**